src/App.js

```diff
@@ -24,6 +32,11 @@ const App = () => (
       <span>Users: </span>
       {users.length}
     </p>
+    <ul className="posts-container">
+      {postsWithUsers.map(post => (
+        <Post key={post.id} {...post} />
```

Comment:
```suggestion
        <Post key={post.id} post={post} />
```
it is better to write this way